packages/wonder-blocks-tooltip/components/tooltip.md

```diff
@@ -34,7 +34,7 @@ const {View, Text} = require("@khanacademy/wonder-blocks-core");
     <div style={{height: 100, overflow: "auto", border: "1px solid", margin: 10,}}>
         <div style={{height: "200vh"}}>
```

Comment:
These should be `View`s.  I'm guessing the rebase didn't come through yet.

Reply:
Could be merge flog. I'll take a look.